ios/FluentUI/Vnext/List/List.swift

```diff
@@ -106,16 +143,24 @@ extension MSFListView {
     struct MSFListCellView: View {
         var cell: MSFListVnextCellData
         var layoutType: MSFListCellVnextLayoutType
+        @State var isExpanded: Bool
         @ObservedObject var tokens: MSFListTokens
 
         init(cell: MSFListVnextCellData, layoutType: MSFListCellVnextLayoutType, tokens: MSFListTokens) {
             self.cell = cell
             self.layoutType = layoutType
             self.tokens = tokens
+            _isExpanded = State(initialValue: cell.isExpanded)
         }
 
         var body: some View {
-            Button(action: cell.onTapAction ?? {}, label: {
+            Button(action: cell.onTapAction ?? {
+                if cell.children != nil {
+                    withAnimation {
+                        isExpanded.toggle()
```

Comment:
This should change the underlying state of the cell to make sure the expanded/collapsed states remain intact.

I the scenario below, the "Subchild #" nodes should remain expanded when I expand Kat Larrson again.


https://user-images.githubusercontent.com/68076145/105077382-4e6be500-5a41-11eb-9d41-a46568d9ca46.mov